ui/guest/src/react/MoveModeZoneMenu.tsx

```diff
@@ -136,13 +148,14 @@ export function MoveModeZoneMenu(props: MoveModeZoneMenuProps) {
 
   useEffect(() => {
     const onClickingOutsideOfSelectedZone = (e: MouseEvent) => {
-      console.log('click');
+      refs.current.onCancel();
     };
+
     window.addEventListener('click', onClickingOutsideOfSelectedZone);
     return () => {
       window.removeEventListener('click', onClickingOutsideOfSelectedZone);
     };
-  }, []);
+  }, [iceId]);
```

Comment:
This effect doesn't use iceId. Why listed as dependency?
```suggestion
  }, []);
```